addons/knative/0.14.x/knative-2.yaml

```diff
@@ -35,4 +35,4 @@ spec:
         customMetricsApiservice:
           enabled: false
         configDeployment:
-          registriesSkippingTagResolving: "gcr.io,docker.io,mcr.microsoft.com,nvcr.io"
+          registriesSkippingTagResolving: "gcr.io,docker.io,index.docker.io,mcr.microsoft.com,nvcr.io"
```

Comment:
Thanks for capturing this issue.

Not yours. Docker diverse in different registry URLs. Could you please add two more?
`registry-1.docker.io`
`registry.hub.docker.com`